{"uuid": "f8d84a7a-bace-406f-868f-bc7048eacd2c", "children": ["a0da0907-b102-4d4b-87da-dfd7692df06c"], "befores": [{"name": "setup_browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: <html>\r\n<head><title>401 Authorization Required</title></head>\r\n<body bgcolor=\"white\">\r\n<center><h1>401 Authorization Required</h1></center>\r\n<hr><center>nginx/1.14.2</center>\r\n</body>\r\n</html>\r\n\n\n", "trace": "  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/conftest.py\", line 45, in setup_browser\n    driver = webdriver.Remote(\n             ^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 209, in check_response\n    raise exception_class(value)\n"}, "start": 1685191138003, "stop": 1685191138306}], "start": 1685191138003, "stop": 1685191138389}